core/trino-main/src/main/java/io/trino/sql/planner/NodePartitioningManager.java

```diff
@@ -235,6 +238,26 @@ private NodePartitionMap systemNodePartitionMap(
         });
     }
 
+    /**
+     * Query plans typically divide data into buckets to help split the work among Trino nodes.  How many buckets?  That is usually dictated by the connector, via
```

Comment:
`usually` - probably not really - but whatever. 
You can rephrase that it is either steered by connector or by engine if connector does not care.

Reply:
Dropped "usually".